tests/datasource/fluent/data_asset/test_data_asset.py

```diff
@@ -1,91 +1,80 @@
+from typing import List
+
 import pytest
 
 from great_expectations.core.batch_config import BatchConfig
-from great_expectations.core.data_context_key import DataContextVariableKey
 from great_expectations.data_context.data_context.abstract_data_context import (
     AbstractDataContext,
 )
-from great_expectations.data_context.store.datasource_store import DatasourceStore
+from great_expectations.data_context.data_context.cloud_data_context import (
+    CloudDataContext,
+)
 from great_expectations.datasource.fluent.interfaces import DataAsset, Datasource
+from great_expectations.datasource.fluent.pandas_datasource import PandasDatasource
 
-
-@pytest.fixture
```

Comment:
Converted all these string fixtures to just be constants for better readability.

Reply:
Nit - could we throw a `Final` annotation on them? I think we do that elsewhere with global constants but not a big deal since the all caps is an understood convention